tests/test_crm.py

```diff
@@ -0,0 +1,517 @@
+from swsscommon import swsscommon
+import os
+import re
+import time
+import json
+import redis
+
+
+def getCrmCounterValue(dvs, key, counter):
+
+    counters_db = swsscommon.DBConnector(swsscommon.COUNTERS_DB, dvs.redis_sock, 0)
+    crm_stats_table = swsscommon.Table(counters_db, 'CRM')
+
+    for k in crm_stats_table.get(key)[1]:
+        if k[0] == counter:
+            return int(k[1])
+
+
+def setReadOnlyAttr(dvs, obj, attr, val):
+
+    db = swsscommon.DBConnector(swsscommon.ASIC_DB, dvs.redis_sock, 0)
+    tbl = swsscommon.Table(db, "ASIC_STATE:{0}".format(obj))
+    keys = tbl.getKeys()
+
+    assert len(keys) == 1
+
+    swVid = keys[0]
+    r = redis.Redis(unix_socket_path=dvs.redis_sock, db=swsscommon.ASIC_DB)
+    swRid = r.hget("VIDTORID", swVid)
+
+    assert swRid is not None
+
+    ntf = swsscommon.NotificationProducer(db, "SAI_VS_UNITTEST_CHANNEL")
+    fvp = swsscommon.FieldValuePairs()
+    ntf.send("enable_unittests", "true", fvp)
+    fvp = swsscommon.FieldValuePairs([(attr, val)])
+    key = "SAI_OBJECT_TYPE_SWITCH:" + swRid
+
+    ntf.send("set_ro", key, fvp)
+
+
+def test_CrmFdbEntry(dvs):
+
+    dvs.runcmd("crm config polling interval 1")
+
+    setReadOnlyAttr(dvs, 'SAI_OBJECT_TYPE_SWITCH', 'SAI_SWITCH_ATTR_AVAILABLE_FDB_ENTRY', '1000')
+
+    time.sleep(5*60)
```

Comment:
do we still need to wait for 300 minutes here? I saw you change the polling interval to 1 second now.

Reply:
@lguohan, first time setting the polling interval will take effect only after the previous polling interval. Future wait would be _1 sec_